hpx/runtime/threads/policies/scheduler_mode.hpp

```diff
@@ -32,15 +32,18 @@ namespace hpx { namespace threads { namespace policies
             ///< scheduler to dynamically increase and reduce the number of
             ///< processing units it runs on. Setting this value not succeed for
             ///< schedulers that do not support this functionality.
-        enable_stealing = 0x20,          ///< This option allows for certain
+        enable_stealing_core = 0x20,          ///< This option allows for certain
             ///< schedulers to explicitly disable thread stealing
-        enable_idle_backoff = 0x40,     ///< This option allows for certain
+        enable_stealing_numa = 0x40,          ///< This option allows for certain
+            ///< schedulers to explicitly disable thread stealing
```

Comment:
These could do with more thorough descriptions. What's the distinction between the two modes? Should they be `enable_work_stealing` and `enable_work_stealing_numa`? Is `enable_stealing_core == false` and `enable_stealing_numa == true` an allowed configuration? And what would it do if it is?

Reply:
`enable_stealing_core = OFF` and `enable_stealing_NUMA=ON` is not really meaningful, but I added `_core` simply a a way of making it clear what was going on in the code to distinguish the two flags.